branding/spacewalk-branding.changes

```diff
@@ -1,3 +1,4 @@
+- Better label visualization when the input is disabled
```

Comment:
Please add the bugnumber to the changelog . For a submit we need at least one. So we should use every chance we have.

Reply:
Done